docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,728 @@
+    | OK                                                   |
+    +------------------------------------------------------+
+    ```
+
+### audit_log_rotate()
+
+#### Parameters
+
+None.
+
+#### Returns
+
+This function returns renamed file name.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_rotate();
+```
+
+
+### audit_log_filter_set_filter(filter_name, definition)
+
+This function, when provided with a filter name and definition, adds the filter. 
+
+The new filter has a different filter ID. Generates an error if the filter name exists.
+
+#### Parameters
+
+* `filter_name` - a selected filter name as a string.
+
+* `definition` - Defines the definition as a JSON value.
+
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SET @filter = '{ "filter_name": { "log": true }}'
+mysql> SET audit_log_filter_set_filter('filter-name', @filter);
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------------------------------+
+    | audit_log_filter_set_filter('filter-name', @filter)  |
+    +-------------------------------------------------------------+
+    | OK                                                          |
+    +-------------------------------------------------------------+
+    ```
+
+### audit_log_filter_set_user(user_name, filter_name)
+
+This function assigns the filter to the selected user account.
+
+A user account can only have one filter. If the user account already has a filter, this function replaces the current filter. If the user account is in a current session, nothing happens. When the user account connects again the new filter is used.
+
+The user name, `%`, is the default account. The filter assigned to `%` is used by any user account without a defined filter.
+
+#### Parameters
+
+* `user_name` - a selected user name in either the `user_name`@`host_name` format or `%`.
+
+* `filter_name` - a selected filter name as a string.
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_set_user('user-name@localhost', 'filter-name');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------------------------------------+
+    | audit_log_filter_set_user('user-name@localhost', 'filter-name')  |
+    +-------------------------------------------------------------------+
+    | OK                                                                |
+    +-------------------------------------------------------------------+
+    ```
+
+## Audit log filter options and variables
+
+| Name |
+| --- |
+| [`audit-log-filter`](#audit-logvalue) |
+| [`audit_log_buffer_size`](#audit_log_buffer_size) |
+| [`audit_log_filter_compression`](#audit_log_filter_filter_compression) |
```

Comment:
audit_log_filter.compression and similar for all the following names

Reply:
fixed